src/ml.cxx

```diff
@@ -0,0 +1,150 @@
+#ifndef GUARD_ML_H
+#define GUARD_ML_H
+
+#include "../include/basefunctions.hxx"
+#include "../include/defaults.hxx"
+#include "../include/utility/Logger.hxx"
+#include "../include/utility/utility.hxx"
+#include "../include/ml.hxx"
+#include "../include/utility/OnnxSessionManager.hxx"
+#include "../include/vectoroperations.hxx"
+#include "ROOT/RDataFrame.hxx"
+#include "ROOT/RVec.hxx"
+#include <Math/Vector4D.h>
+#include <Math/VectorUtil.h>
+
+#include "TMVA/RModel.hxx"
+#include "TInterpreter.h"
+#include "TMVA/RModelParser_ONNX.hxx"
+#include "TSystem.h"
+#include <memory>
+#include <onnxruntime_cxx_api.h>
+#include <assert.h>
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <nlohmann/json.hpp>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <string>
+
+using json = nlohmann::json;
+
+
+namespace ml {
+/**
+ * @brief Function to perform a standard transformation of input variables for
+ * NN evaluation.
+ *
+ * @param df The input dataframe
+ * @param inputname name of the variable which should be transformed
+ * @param outputname name of the output column
+ * @param param_file path to a json file with a dictionary of mean and std
+ * values
+ * @param var_type variable data type for correct processing e.g. "i" for
+ * integer or "f" for float
+ * @return a new dataframe containing the new column
+ */
+ROOT::RDF::RNode StandardTransformer(ROOT::RDF::RNode df,
```

Comment:
This producer is also trimmed for even/odd evaluation and I also changed the format of the input information file. While I used json files, in https://github.com/KIT-CMS/CROWN/pull/235 csv files are used. So I think either a common definition should be defined for CROWN or the producer needs to be expanded to be more generic and able to take different input formats.

Reply:
We could also rename it to json in the name and the other implementation to csv in the name to have multiple functions available. I am not a fan of depending on something like file endings